scripts/onboarding/aks/onboarding-msi-terraform-syslog/main.tf

```diff
@@ -0,0 +1,75 @@
+resource "azurerm_resource_group" "rg" {
+  name     = var.aks_resource_group_name
+  location = var.resource_group_location
+}
+
+resource "azurerm_kubernetes_cluster" "k8s" {
+  name                = var.cluster_name
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  dns_prefix          = var.dns_prefix
+
+  tags = var.resource_tag_values
+  
+  default_node_pool {
+    name       = "agentpool"
+    vm_size    = "Standard_D2_v2"
+    node_count = var.agent_count
+  }
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  oms_agent {
+    log_analytics_workspace_id = var.workspace_resource_id
+    msi_auth_for_monitoring_enabled = true
+  }
+
+  network_profile {
+    network_plugin    = "kubenet"
+    load_balancer_sku = "standard"
+  }
+}
+
+resource "azurerm_monitor_data_collection_rule" "dcr" {
+  name                = "MSCI-dcr-${var.workspace_region}-${var.cluster_name}"
```

Comment:
Follow the same naming as in AZURE CLI and ARM template.

Reply:
changed to remove the 'dcr' word in the middle